src/v3/src/types/schema.ts

```diff
@@ -174,12 +176,13 @@ export interface ButtonElement extends UISchemaElement {
 export interface AuthenticatorButtonElement {
```

Comment:
I updated this to make options extend ButtonElement['options'], originally, we had to exclude some props from ButtonElement because AuthenticatorButtonElement didnt use them but now it uses them all. I could not extend AuthenticatorButtonElement with ButtonElement at the interface because the `type` value differs between them and I receive a TS error when I tried it.